binstar_client/utils/detect.py

```diff
@@ -48,6 +48,20 @@ def is_ipynb(filename):
     log.debug("No ipynb file")
 
 
+def is_project(filename):
+    log.debug("Testing if project ..")
+
+    def is_python_file():
```

Comment:
what is the use of defining a function here? maybe to add logs? I only see unnecessary complexity.